src/DynamoCoreWpf/UI/GuidedTour/CustomRichTextBox.cs

```diff
@@ -69,113 +72,124 @@ private static FlowDocument GetCustomDocument(string Text)
-                }
-                else if (bImageActive)
-                {
-                    //this will contatenate each word so at the end we will have the full image path
-                    imageName += word;
-                }
-                else if (bBulletListActive == true)
-                {
-                    bulletEntryText += (word + " ");
-                }
-                else
-                {
-                    para.Inlines.Add(word);
-                }
+                    {
+                        para.Inlines.Add(word);
+                    }
 
-                //End of the text with bold formatting.
-                if (word.Trim().EndsWith("**"))
-                {
-                    bBoldActive = false;
-                }
-                //End of the image between text formatting.
-                if (word.EndsWith("%"))
-                {
-                    bImageActive = false;
-
-                    var tmpImage = new Image();
-                    tmpImage.Source = new BitmapImage(new Uri(imageName.Replace("%", ""), UriKind.RelativeOrAbsolute));
-                    imageContainer.Child = tmpImage;
-                    Figure imageFig = new Figure(imageContainer);
-                    imageFig.Width = new FigureLength(30);
-                    imageFig.Height = new FigureLength(30);
-                    para.Inlines.Add(imageFig);
+                    //End of the text with bold formatting.
+                    if (word.Trim().EndsWith("**"))
+                    {
+                        bBoldActive = false;
+                    }
+                    //End of the image between text formatting.
+                    if (word.EndsWith("%"))
+                    {
+                        bImageActive = false;
+
+                        var tmpImage = new Image();
+                        tmpImage.Source = new BitmapImage(new Uri(imageName.Replace("%", ""), UriKind.RelativeOrAbsolute));
+                        imageContainer.Child = tmpImage;
+                        Figure imageFig = new Figure(imageContainer);
+                        imageFig.Width = new FigureLength(30);
+                        imageFig.Height = new FigureLength(30);
+                        para.Inlines.Add(imageFig);
+                    }
+                    //End fo the Bullet Items formatting
+                    if (word.EndsWith("-"))
+                    {
+                        bBulletListActive = false;
+                        bulletedItemsList.ListItems.Add(new ListItem(new Paragraph(new Run(bulletEntryText.Replace("-", "")))));
+                        bulletEntryText = string.Empty;
+                    }
+
+                    para.Inlines.Add(" ");
                 }
-                //End fo the Bullet Items formatting
-                if (word.EndsWith("-"))
+                var Text1 = String.Join(String.Empty, para.Inlines.Select(line => line.ContentStart.GetTextInRun(LogicalDirection.Forward)));
```

Comment:
Where is this `Text1` used?

Reply:
@QilongTang it was not used, then it was removed in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12250/commits/c3995980180bb91f906ebbe6eb891924d0a584ea